code/modules/cargo/packs.dm

```diff
@@ -1077,6 +1123,41 @@
 	)
 	crate_name = "portable pump crate"
 	crate_type = /obj/structure/closet/crate/large
+
+/datum/supply_pack/engineering/cones
+	name = "Engineering Hat Crate"
+	desc = "Contains a complete set of headwear to fit the heads of an entire engineering crew."
+	cost = 600
+	contraband = TRUE
+	contains = list(
+		/obj/item/clothing/head/cone,
+		/obj/item/clothing/head/cone,
+		/obj/item/clothing/head/cone,
+		/obj/item/clothing/head/cone,
+		/obj/item/clothing/head/cone,
+		/obj/item/clothing/head/cone
+	)
+	crate_name = "engineering hat crate"
+
+/datum/supply_pack/engineering/radios
+	name = "Telecommunications Starter Pack Crate"
+	desc = "Contains everything you need to get a basic stationwide radio network established!"
+	cost = 500
+	contraband = TRUE
```

Comment:
Should it be contraband?

Reply:
For the radios they're mostly a joke crate with a bad description, which is why I labelled it contraband.